dev/VSIX/Shared/WizardImplementation.cs

```diff
@@ -13,161 +13,159 @@
 
-    public class NuGetPackageInstaller : IWizard
+    public void RunStarted(object automationObject, Dictionary<string, string> replacementsDictionary, WizardRunKind runKind, object[] customParams)
     {
-        internal static Guid SolutionVCProjectGuid = new Guid("8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942");
-        private Project _project;
-        private IComponentModel _componentModel;
-        private IEnumerable<string> _nuGetPackages;
-        private IVsNuGetProjectUpdateEvents _nugetProjectUpdateEvents;
-        private IVsThreadedWaitDialog2 _waitDialog;
-
-        public void RunStarted(object automationObject, Dictionary<string, string> replacementsDictionary, WizardRunKind runKind, object[] customParams)
+        ThreadHelper.ThrowIfNotOnUIThread();
+        _componentModel = (IComponentModel)ServiceProvider.GlobalProvider.GetService(typeof(SComponentModel));
+        _waitDialog = ServiceProvider.GlobalProvider.GetService(typeof(SVsThreadedWaitDialog)) as IVsThreadedWaitDialog2;
+        if (_componentModel != null)
         {
-            ThreadHelper.ThrowIfNotOnUIThread();
-            _componentModel = (IComponentModel)ServiceProvider.GlobalProvider.GetService(typeof(SComponentModel));
-            _waitDialog = ServiceProvider.GlobalProvider.GetService(typeof(SVsThreadedWaitDialog)) as IVsThreadedWaitDialog2;
-            if (_componentModel != null)
-            {
-                _nugetProjectUpdateEvents = _componentModel.GetService<IVsNuGetProjectUpdateEvents>();
-                if (_nugetProjectUpdateEvents != null)
-                {
-                    _nugetProjectUpdateEvents.SolutionRestoreFinished += OnSolutionRestoreFinished;
-                }
-            }
-            // Assuming package list is passed via a custom parameter in the .vstemplate file
-            if (replacementsDictionary.TryGetValue("$NuGetPackages$", out string packages))
+            _nugetProjectUpdateEvents = _componentModel.GetService<IVsNuGetProjectUpdateEvents>();
+            if (_nugetProjectUpdateEvents != null)
             {
-                _nuGetPackages = packages.Split(';').Where(p => !string.IsNullOrEmpty(p));
+                _nugetProjectUpdateEvents.SolutionRestoreFinished += OnSolutionRestoreFinished;
             }
-        }        
-        public void ProjectFinishedGenerating(Project project)
+        }
+        // Assuming package list is passed via a custom parameter in the .vstemplate file
+        if (replacementsDictionary.TryGetValue("$NuGetPackages$", out string packages))
         {
-            ThreadHelper.ThrowIfNotOnUIThread();
-            _project = project;
-            Guid _projectGuid;
-            if (project != null)
-            {
-                Guid.TryParse(project.Kind, out _projectGuid);
+            _nuGetPackages = packages.Split(';').Where(p => !string.IsNullOrEmpty(p));
+        }
+    }        
+    public void ProjectFinishedGenerating(Project project)
```

Comment:
```suggestion
    }
    
    public void ProjectFinishedGenerating(Project project)
```
nit: newline between functions

Reply:
newline added